ledger/sandbox-classic/src/test/lib/scala/platform/sandbox/services/SandboxFixture.scala

```diff
@@ -20,7 +20,6 @@ trait SandboxFixture extends AbstractSandboxFixture with SuiteResource[(SandboxS
 
   override protected def config: SandboxConfig =
     super.config.copy(
-      seeding = None,
```

Comment:
Don't you want it to set it to "weak" for testing?

Reply:
I think that 
`super.config.seeding = Some(Seeding.Weak)` already.